stream_alert/shared/lookup_tables/drivers.py

```diff
@@ -0,0 +1,146 @@
+from abc import abstractmethod, ABCMeta
+from stream_alert.shared.logger import get_logger
+from stream_alert.shared.lookup_tables.errors import LookupTablesConfigurationError
+
+LOGGER = get_logger(__name__)
+
+
+# pylint: disable=invalid-name
```

Comment:
why is this name invalid?

Reply:
Bad copypasta

Reply:
Wait... hold on it's actually valid; there's a property named `id` that I was accidentally catching.

I'll move onto the prop